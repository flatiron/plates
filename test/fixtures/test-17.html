<ul>
  <li class="organizations">
    <h3 class="name"></h3>
    <ul>
      <li class="staff">
        <table>
          <tr class="name">
            <td class="first"></td>
            <td class="last"></td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
</ul>
