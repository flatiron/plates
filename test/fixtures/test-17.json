{
  "organizations" : [
    {
      "name": "DotOrg",
      "staff": [
        {
          "name": {
            "first": "Dot",
            "last" : "Matrix"
          }
        },
        {
          "name": {
            "first": "Serious",
            "last" : "Business"
          }
        }
      ]
    },
    {
      "name": "FooBar",
      "staff": [
        {
          "name": {
            "first": "Foo",
            "last" : "Bar"
          }
        },
        {
          "name": {
            "first": "Baz",
            "last" : "Bitsom"
          }
        }
      ]
    }
  ]
}
