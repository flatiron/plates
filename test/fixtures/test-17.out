<ul>
  <li class="organizations">
    <h3 class="name">DotOrg</h3>
    <ul>
      <li class="staff">
        <table>
          <tr class="name">
            <td class="first">Dot</td>
            <td class="last">Matrix</td>
          </tr>
        </table>
      </li><li class="staff">
        <table>
          <tr class="name">
            <td class="first">Serious</td>
            <td class="last">Business</td>
          </tr>
        </table>
      </li>
    </ul>
  </li><li class="organizations">
    <h3 class="name">FooBar</h3>
    <ul>
      <li class="staff">
        <table>
          <tr class="name">
            <td class="first">Foo</td>
            <td class="last">Bar</td>
          </tr>
        </table>
      </li><li class="staff">
        <table>
          <tr class="name">
            <td class="first">Baz</td>
            <td class="last">Bitsom</td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
</ul>
